examples/buildkit0/buildkit.go

```diff
@@ -24,7 +24,7 @@ func main() {
 	bk := buildkit(opt)
 	out := bk.Run(llb.Shlex("ls -l /bin")) // debug output
 
-	dt, err := out.Marshal()
+	dt, err := out.Marshal(llb.LinuxAmd64)
```

Comment:
LGTM but probably we should detect OS and Arch?

Reply:
Detection is the default behavior. As these examples generate LLB in a different process than the client it is possible that the platforms do not match. Eg. running the example on macos and then sending the output to `buildctl build`.